### PR TITLE
Makes a popover in a modal responsive

### PR DESCRIPTION
**Pull request by Manasa2850**
## What this PR does
< vertical layout is present at all times >

Credits for initial code : @codingmamakaz

## Screenshots
Before:
Before

After:
After

### CODE REVIEW

**Comment by ragesoss** [comment]
Hmm... I think this will need to be scoped specifically to that one popover.

It causes a regression in others, like the AssignButton one, where the `en.wikipedia.org (change)` part gets shifted to the same line as the input, instead of below it like it should be.

regression

**Comment by ragesoss** [comment]
Also, please add credit for the person whose work you're starting from in the commit message.

**Comment by Manasa2850** [comment]
@ragesoss sorry I'll add the credits immediately.

**Comment by Manasa2850** [comment]
@ragesoss sure, I'll make changes to just that one popover.

**Comment by ragesoss** [comment]
@Manasa2850 are you still working on this?

**Comment by Manasa2850** [comment]
@ragesoss yes I'm working on this. Sorry I was a little busy because of my tests.

**Comment by Manasa2850** [comment]
I will try to resolve this by tomorrow night.

**Comment by Manasa2850** [comment]
@ragesoss I've created another class 'modal-course-details1' which can be called whenever there is an issue with the popover. Hope this is fine

**Comment by Manasa2850** [comment]
@ragesoss I've solved it specifically for that one popover. Hope it's fine

File: app/assets/stylesheets/modules/_course.styl [hunk 1]
```diff
@@ -210,4 +210,17 @@ rating(color)
...
\ No newline at end of file
+  display: none
+
+.modal-course-details
+    .pop__container
+        form
+          display flex
+          align-items center
+          justify-content center
+        .pop.open
```
Comment (ragesoss) [inline app/assets/stylesheets/modules/_course.styl]: Why is this styling only in the `open` mode?

It looks like this is causing a problem with the transition animation: the modal shifts out of column format for a brief flash as the popover closes.

**Comment by Manasa2850** [comment]
@ragesoss is it fine now?

**Comment by ragesoss** [comment]
Looks good to me.

**Pull request merged**